packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -235,6 +232,8 @@ class PouchLink extends CozyLink {
       }
     }
 
+    this.queryEngine = this.options.platform?.queryEngine || PouchDBQueryEngine
```

Comment:
This one seems not used anymore, you can directly use the platform member inside of PouchManager.

Also PouchManager already defaults to `PouchDBQueryEngine`, so no need to handle it here.